src/pdf/base/XojCairoPdfExport.cpp

```diff
@@ -143,7 +144,8 @@ void XojCairoPdfExport::exportPageLayers(size_t page) {
     for (const auto& layer: *p->getLayers()) layer->setVisible(initialVisibility[layer]);
 }
 
-auto XojCairoPdfExport::createPdf(fs::path const& file, PageRangeVector& range, bool progressiveMode) -> bool {
+auto XojCairoPdfExport::createPdf(fs::path const& file, const std::vector<PageRangeEntry>& range, bool progressiveMode)
+        -> bool {
```

Comment:
Why not declare this function as
```cpp
bool XojCairoPdfExport::createPdf(/*...*/)  {/*...*/}
```
?

I find this syntax a bit odd, but maybe it's just me.

Reply:
There are voices in the developer community that see this style as the future. I'm not quite sure myself if it really will be but we'll see. For now we do not restrict it that heavily, though we might take that step in the future.